Source/Scene/Cesium3DTile.js

```diff
@@ -212,7 +235,9 @@ define([
          * @readonly
          */
         orientedBoundingBox : {
+            //TODO deal with this property
```

Comment:
It's probably best to remove this getter. I added it as a way to get the bounding volume for `Instanced` and `Points` content providers. I'll have a small commit handling this soon.